nixos/modules/services/databases/openldap.nix

```diff
@@ -53,6 +53,13 @@ in
         description = "The database directory.";
       };
 
+      configDir = mkOption {
+        type = types.string;
```

Comment:
`string` is an obsolete type. Consider `path` instead, it will enforce that the name begins with a slash (i.e., that it looks somewhat like a file path).